subprojects/language-java/src/integTest/groovy/org/gradle/javadoc/JavadocIntegrationTest.groovy

```diff
@@ -25,10 +25,12 @@ import org.junit.Rule
 import spock.lang.Issue
 
 class JavadocIntegrationTest extends AbstractIntegrationSpec {
-    @Rule TestResources testResources = new TestResources(temporaryFolder)
+    @Rule
```

Comment:
⭕ Either revert auto-formatting or adjust the change (i.e. add a newline between the `when` and `then` statements).